### ISSUE TITLE
[Documentation] Service file path should start from /etc/systemd/system

### ISSUE THREAD

**Issue by LukasDoesDev** [Toqozz/wired-notify#70]
`man 7 file-hierarchy`
```
GENERAL STRUCTURE
(...)
       /etc/
           System-specific configuration. This directory may or may not be read-only. Frequently, this directory is pre-populated with vendor-supplied configuration files, but
           applications should not make assumptions about this directory being fully populated or populated at all, and should fall back to defaults if configuration is missing.
(...)
VENDOR-SUPPLIED OPERATING SYSTEM RESOURCES
       /usr/
           Vendor-supplied operating system resources. Usually read-only, but this is not required. Possibly shared between multiple hosts. This directory should not be modified by
           the administrator, except when installing or removing vendor-supplied packages.
```

`man 5 systemd.unit`
```man
UNIT FILE LOAD PATH
(...)
       Table 1.  Load path when running in system mode (--system).
       ┌──────────────────────────────┬────────────────────────────────────────────────────────────┐
       │Path                          │ Description                                                │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/etc/systemd/system.control   │ Persistent and transient configuration created using the   │
       ├──────────────────────────────┤ dbus API                                                   │
       │/run/systemd/system.control   │                                                            │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/run/systemd/transient        │ Dynamic configuration for transient units                  │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/run/systemd/generator.early  │ Generated units with high priority (see early-dir in       │
       │                              │ systemd.generator(7))                                      │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/etc/systemd/system           │ System units created by the administrator                  │ <----- Here
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/run/systemd/system           │ Runtime units                                              │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/run/systemd/generator        │ Generated units with medium priority (see normal-dir in    │
       │                              │ systemd.generator(7))                                      │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/usr/local/lib/systemd/system │ System units installed by the administrator                │
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/usr/lib/systemd/system       │ System units installed by the distribution package manager │ <----- Not here
       ├──────────────────────────────┼────────────────────────────────────────────────────────────┤
       │/run/systemd/generator.late   │ Generated units with low priority (see late-dir in         │
       │                              │ systemd.generator(7))                                      │
       └──────────────────────────────┴────────────────────────────────────────────────────────────┘
```

TL;DR: `/etc/systemd/system` should have units made by the system administrator and `/usr/lib/systemd/system` should have units from packages.

**Comment by Toqozz** [comment]
Wow this path stuff is confusing.  Basically, because we're telling users to copy the file instead of installing it ourselves, it should go in `/etc/systemd/system` instead.

But wait, this sounds more like a service being installed by the admin rather than created; should it be `usr/local/lib/systemd/system`?

**Comment by LukasDoesDev** [comment]
> Wow this path stuff is confusing.  Basically, because we're telling users to copy the file instead of installing it ourselves, it should go in `/etc/systemd/system` instead.
> 
> But wait, this sounds more like a service being installed by the admin rather than created; should it instead be `usr/local/lib/systemd/system`?

I guess yeah it could be installed there too, but I've found myself modifying the file to customize the wired-notify command.

**Comment by Toqozz** [comment]
Fixed in README now.  Thanks for the issue!
